lib/src/specs/class.dart

```diff
@@ -75,6 +101,15 @@ abstract class ClassBuilder extends Object
   /// Whether the class is `abstract`.
   bool abstract = false;
 
+  /// Whether the class is `sealed`.
+  bool sealed = false;
```

Comment:
Is there a reason that `sealed` isn't modeled as a `ClassModifier`?

Reply:
cc @srawlins - I'm curious if you have any thoughts on how to model class modifiers - is it worth trying to encode _some_ of the cross usage constraints by having some be an `enum`, or do you think we should model each modifier as a boolean?

Reply:
How this is implemented now works out pretty nicely, looking at [this table from the spec](https://github.com/dart-lang/language/blob/main/accepted/future-releases/class-modifiers/feature-specification.md#syntax).

This impl does encode a bit what is legal and what is not.

Building from scratch, it would probably be simplest to have one enum for all of the modifiers, and then store a List (or Set) of them. But that would not be backwards compatible?

I don't really have a strong opinion.